### PR TITLE
Implemented driver for Pheanstalk 4

### PR DESCRIPTION
**Pull request by SamMousa**
Implements #1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "codeception/codeception": "4.0.x-dev | ^4.0"
     },
     "require-dev": {
-        "pda/pheanstalk": "~3.0",
+        "pda/pheanstalk": "> 3",
```
Comment (Naktibalda) [inline composer.json]: Lower boundary is not clear and upper boundary is not set.
I think that `"^3.0 | ^4.0"` would be better.

Comment (Naktibalda) [reply composer.json]: P.S. Please update comment in "suggest" which says thay v4 is not supported.

File: .travis.yml [hunk 2]
```diff
@@ -9,11 +9,17 @@ php:
...
+    packages:
+      - beanstalkd
 install:
   - '[[ -z "$CI_USER_TOKEN" ]] || composer config github-oauth.github.com ${CI_USER_TOKEN};'
   - travis_retry composer self-update && composer --version
   - travis_retry composer update --prefer-dist --no-interaction
 
 script:
   - php ./vendor/bin/codecept run
+  - if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then php ./vendor/bin/codecept run --coverage-xml; fi
```
Comment (Naktibalda) [inline .travis.yml:23]: Test fails on second run because the number of records is twice as large as expected.

**Review by Naktibalda — CHANGES_REQUESTED** [review]

**Comment by SamMousa** [comment]
Ah, my bad, assumed tests didn't have side effects

**Comment by Naktibalda** [comment]
@SamMousa are you going to fix it?

**Comment by SamMousa** [comment]
Yes, thanks for the ping. Put it on my list!

**Comment by SamMousa** [comment]
@Naktibalda ping!

File: .travis.yml (hunk 2 above)
Comment (SamMousa) [reply .travis.yml:23]: Fixed it by checking the initial state instead of assuming it.

**Pull request merged**